README.md

```diff
@@ -110,9 +139,9 @@ build distinct package versions.
 
 In order to produce a uniquely identifiable distribution:
  * If the version of a package **is not** being increased, please add or increase
-   the [``build/number``](https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#build-number-and-string).
+   the [``build/number``](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#build-number-and-string).
  * If the version of a package **is** being increased, please remember to return
-   the [``build/number``](https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#build-number-and-string)
+   the [``build/number``](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#build-number-and-string)
    back to 0.
 
 Feedstock Maintainers
```

Comment:
I think it'll be a good idea for you to add your name as well here. =)

Reply:
Agree

Reply:
The proper way of adding a person to the maintainer list is given here https://conda-forge.org/docs/maintainer/updating_pkgs/#updating-the-maintainer-list. Once this PR is merged I can start the process by creating an issue, then all that's needed is for one of the current maintainers to accept the PR that will be generated.

Reply:
Gotcha, it's been a while, and I forgot the process. But yes, after this PR let's get you on the list of the maintainers.